LSP-rust-analyzer.sublime-settings

```diff
@@ -2,6 +2,16 @@
 	"selector": "source.rust",
 	"command": ["${storage_path}/LSP-rust-analyzer/rust-analyzer"],
 	"settings": {
+
+		// Settings Not Related to rust-analyzer Server
+
+		// Wether or not to automatically close terminus panel/tab upon completion.
+		"rust-analyzer.terminusAutoClose": false,
+		// Wether or not to spawn a panel at the bottom, or a new tab
+		"rust-analyzer.terminusUsePanel": false,
```

Comment:
The current implementation of `get_setting` won't read settings from this place, will it? You are reading from the root object in this file.

Reply:
oh lemme see